Extensions/3D/Cube3DRuntimeObjectPixiRenderer.ts

```diff
@@ -75,14 +80,37 @@ namespace gdjs {
     ) {
       const geometry = new THREE.BoxGeometry(1, 1, 1);
       // TODO (3D) - feature: support color instead of texture?
-      const materials = [
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[0]),
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[1]),
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[2]),
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[3]),
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[4]),
-        getFaceMaterial(runtimeObject, materialIndexToFaceIndex[5]),
-      ];
+      const materials: THREE.Material[] = [];
+
+      for (let i = 0; i < 6; i++) {
```

Comment:
Good idea to use a loop!
In JS, there's a more elegant way to do this, with a `map`.
So you could write:

```js
new Array(6).fill(0).map((_, index) => {
  return material;
})
```